src/containers/NewProject/NewProject.js

```diff
@@ -1,30 +1,127 @@
 import React from 'react';
-import { Button, Modal, Form, Input } from 'antd';
+import { Button, Modal, Form, Input, Steps, message, Menu, Dropdown, Icon, Card, Table, Divider, Tag } from 'antd';
 import './NewProject.css';
 
+
+const Step = Steps.Step;
+
 const ProjectForm = Form.create({ name: 'form_in_modal' })(
   class extends React.Component {
+
+    renderSteps = (getFieldDecorator, current, next, prev, rooms, addRoom, onCreate, name) => {
+
+      const onClick = function ({ key }) {
+        message.info(`Added a ${key}`);
+        addRoom(key)
```

Comment:
This is adding the clicked rooms to NewProject's state and giving the window notification.